packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -59,13 +59,19 @@ import {
   LoopbackLocation,
```

Comment:
Can we verify that these tests update the state accordingly? https://github.com/MetaMask/snaps-monorepo/blob/1d7928bd328204570c8594f0ed2403d5344e8aa9/packages/snaps-controllers/src/snaps/SnapController.test.ts#L788 https://github.com/MetaMask/snaps-monorepo/blob/1d7928bd328204570c8594f0ed2403d5344e8aa9/packages/snaps-controllers/src/snaps/SnapController.test.ts#L2202